obkvrpc/connection_pool.go

```diff
@@ -150,6 +161,18 @@ func (p *ConnectionPool) CreateConnection(ctx context.Context) (*Connection, err
 	if err != nil {
 		return nil, errors.WithMessage(err, "connection connect")
 	}
+
+	// listen receive channel
+	go func() {
+		connection.receivePacket()
+		p.disConnectedCount.Add(1) // receivePacket() break means disconnected
```

Comment:
Why receivePacket() break means disconnected? The connection seems to be established in the previous step.

Reply:
connection.receivePacket() is a while(1), only disconnected will break